test/controllers/managing.officer.corporate.controller.spec.ts

```diff
@@ -70,13 +74,20 @@ import {
 import { logger } from "../../src/utils/logger";
 import { hasBeneficialOwnersStatement } from "../../src/middleware/navigation/has.beneficial.owners.statement.middleware";
 import { EMAIL_ADDRESS } from "../__mocks__/session.mock";
+import { ResignedOnDateKey, StartDateKey } from '../../src/model/date.model';
+import { MO_RESIGNED_ON_DATE, MO_START_DATE } from '../__mocks__/fields/date.mock';
 
 const mockHasBeneficialOwnersStatementMiddleware = hasBeneficialOwnersStatement as jest.Mock;
 mockHasBeneficialOwnersStatementMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
 
 const mockAuthenticationMiddleware = authentication as jest.Mock;
 mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next());
 
+const mockMapDataObjectToFields = mapDataObjectToFields as jest.Mock;
```

Comment:
what's the purpose of the mock return value being what the actual method would do? Is that because Jest has a mock on the whole file you needed to do that?

Reply:
Yes this was why. Although tests have been removed from this suite as per your other comment so no longer relevant